webapp/test/components/domain/AdditionalBeaconUseSummary.test.tsx

```diff
@@ -9,7 +9,16 @@ import { BeaconUse } from "../../../src/entities/BeaconUse";
 import { Environment } from "../../../src/lib/deprecatedRegistration/types";
 import { getMockUse } from "../../mocks";
 
+jest.mock("next/router", () => ({
```

Comment:
A little confused by this, what's the reasoning for this add?

Reply:
I used the useRouter next hook to get the current route so I could hide the change links on the main summary page. The tests for this component were failing because the return value of this hook was missing, hence the mock for it.